ouroboros-network/src/Ouroboros/Network/Mux/Types.hs

```diff
@@ -181,14 +229,27 @@ data PerMuxSharedState ptcl m = PerMuxSS {
     dispatchTable :: MiniProtocolDispatch ptcl m
   -- | Egress queue, shared by all miniprotocols
   , tsrQueue      :: TBQueue m (TranslocationServiceRequest ptcl m)
-  -- | Timestamp and send MuxSDU
-  , write         :: MuxSDU ptcl -> m (Time m)
-  -- | Read a MuxSDU
-  , read          :: m (MuxSDU ptcl, Time m)
-  -- | Return a suitable MuxSDU payload size
-  , sduSize       :: m Word16
+  , bearer        :: MuxBearer ptcl m
   }
 
+data MuxBearerState = Larval
```

Comment:
It would be good to give short prose descriptions of these states.

Reply:
Agreed.